### PR TITLE
Add housing unit by value chart

### PR DESCRIPTION
**Pull request by shifucun**
Also make the aggregation function general.

image

### CODE REVIEW

File: server/chart_config.json [hunk 1]
```diff
@@ -937,6 +937,46 @@
       "unit": "%"
     }
   },
+  {
+    "category": "Housing",
+    "title": "Housing Unit Value",
```
Comment (beets) [inline server/chart_config.json:942]: Housing Units by Value

File: server/chart_config.json [hunk 2]
```diff
@@ -937,6 +937,46 @@
...
+      "Count_HousingUnit_HomeValue250000To299999USDollar",
+      "Count_HousingUnit_HomeValue300000To399999USDollar",
+      "Count_HousingUnit_HomeValue400000To499999USDollar",
+      "Count_HousingUnit_HomeValue500000To749999USDollar",
+      "Count_HousingUnit_HomeValue750000To999999USDollar",
+      "Count_HousingUnit_HomeValue1000000To1499999USDollar",
+      "Count_HousingUnit_HomeValue1500000To1999999USDollar",
+      "Count_HousingUnit_HomeValue2000000OrMoreUSDollar"
+    ],
+    "aggregate": "Count_HousingUnit",
```
Comment (beets) [inline server/chart_config.json:971]: In this case, should it be HomeValue? I imagine there are other measurements of housing units too.

File: static/js/shared/stats_var_labels.ts [hunk 3]
```diff
@@ -290,6 +290,17 @@ const STATS_VAR_LABEL: { [key: string]: string } = {
   "WorldBank/SH_SVR_WAST_MA_ZS": "Male",
 
   "WorldBank/SI_POV_GINI": "Gini Index",
+
+  // Housing
+  Count_HousingUnit_HomeValueUpto49999USDollar: "0 - 50k",
```
Comment (beets) [inline static/js/shared/stats_var_labels.ts:295]: Should we add the unit to these labels? 0 - $50k

File: static/js/shared/stats_var_titles.ts [hunk 4]
```diff
@@ -382,6 +382,19 @@ const STATS_VAR_TITLES: { [key: string]: string } = {
   "WorldBank/SH_SVR_WAST_MA_ZS": "Severe Wasting among Children under 5 (Male)",
 
   "WorldBank/SI_POV_GINI": "Gini Index",
+
+  // Housing
+  Count_HousingUnit_HomeValueUpto49999USDollar: "Less than 50,000 $",
+  Count_HousingUnit_HomeValue50000To99999USDollar: "50,000 - 99,999 $",
```
Comment (beets) [inline static/js/shared/stats_var_titles.ts:388]: Should these be $50,000 - $99,999 ?

File: server/chart_config.json [hunk 5]
```diff
@@ -937,6 +937,46 @@
...
+    "relatedChart": {
+      "title": "Housing Unit Percent by Value",
```
Comment (beets) [inline server/chart_config.json:973]: maybe Percentage of Housing Units by Value

**Review by beets — APPROVED** [review]
very cool

File: server/chart_config.json (hunk 1 above)
Comment (shifucun) [reply server/chart_config.json]: Done. Add change other places to "Units"

File: server/chart_config.json (hunk 2 above)
Comment (shifucun) [reply server/chart_config.json]: Done. Making this the "property" to aggregate and added test

File: server/chart_config.json (hunk 5 above)
Comment (shifucun) [reply server/chart_config.json]: Done

File: static/js/shared/stats_var_labels.ts (hunk 3 above)
Comment (shifucun) [reply static/js/shared/stats_var_labels.ts]: Done

File: static/js/shared/stats_var_titles.ts (hunk 4 above)
Comment (shifucun) [reply static/js/shared/stats_var_titles.ts]: Done. Also use K and M here.

**Pull request merged**